core/drivers/gic.c

```diff
@@ -368,3 +404,41 @@ static void gic_op_disable(struct itr_chip *chip, size_t it)
 
 	gic_it_disable(gd, it);
 }
+
+static int gic_op_raise_pi(struct itr_chip *chip, size_t it)
+{
+	struct gic_data *gd = container_of(chip, struct gic_data, chip);
+
+	if (it < 16) {
```

Comment:
Given that we we assert most other parameters in this API shouldn't we do the same here? The rationale is that eventual errors are static programming errors so the recovery action may be limited.